cassandra/cassandra_test.go

```diff
@@ -0,0 +1,279 @@
+package cassandra
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/gocql/gocql"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	cassandracontainer "github.com/testcontainers/testcontainers-go/modules/cassandra"
```

Comment:
suggestion: no need to alias
```suggestion
	"github.com/testcontainers/testcontainers-go/modules/cassandra"
```